docs/docs/how-to/cloud/outgoing-notifications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Outgoing Notifications"
+description: "Learn how to set outgoing notifications based on your build and deploy status"
+---
+
+When your Gatsby site is built and deployed on Gatsby Cloud, you now have the option to hook up a set of outgoing notifications regarding the status of that build. You can send notifications via:
+
+1. A generic webhook.
+1. A webhook that sends a Slack message.
+1. Pull Request comments in GitHub or GitLab.
+
+By default, the Pull Request comments are already set up for existing and new sites in Gatsby Cloud.
+
+### Adding webhook notifications
+
+1. First, navigate to Site Settings. In the sidebar menu under "General", select **Outgoing Notifications**. Then click **Add Notification** and choose the Webhooks option.
```

Comment:
(This also applies to the other docs)

Similarly to my comment in https://github.com/gatsbyjs/gatsby/pull/36218#discussion_r934198384 we should keep it consistent how we denote buttons/things to click in the UI.

I'd change the sentence to this:

```suggestion
1. First, navigate to "Site Settings". In the sidebar menu under "General", select **"Outgoing Notifications"**. Then click **"Add Notification"** and choose the "Webhooks" option.
```

So use bolding where necessary and use `""` to denote the buttons. This is how it was done in https://github.com/gatsbyjs/gatsby/blob/aeae8f366a48a1ec7832dc61b70b719ec7f92cef/docs/docs/how-to/cloud/adding-a-custom-domain.md after my comments